packages/ember-data/lib/system/store.js

```diff
@@ -250,7 +252,8 @@ Store = Ember.Object.extend({
 
     if (DS.Adapter.detect(adapter)) {
```

Comment:
hum, ok, I'm probably missing something, I thought the lookup returns an instance, not a class.